doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -0,0 +1,249 @@
+              Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
+      00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
+              Subsystem: Intel Corporation Ethernet Connection I219-LM
+
+   .. note:: 
+
+      Double check PCI devices BDF defined in the
+      ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``
+      with the information reported by the ``lspci -vv`` command.
+
+#. Clone the ACRN source code and configure the build options
+
+   Please refer :ref:`getting-started-building` to setup ACRN build environment
+   on your development workstation.
+
+   Clone the ACRN source code:
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor/hypervisor
+
+   Configure the build options:
+
+   .. code-block:: none
+   
+      $ make defconfig BOARD=nuc7i7bnh
+      $ make menuconfig
+
+   Updates the following configure item:
+
+     *  Set ACRN Scenario as "Logical Partition VMs";
+     *  Set Maximum number of VCPUs per VM as "2";
+     *  Set Maximum number of PCPU as "4";
+     *  Clear/Disable “Enable hypervisor relocation”.
+
+   We recommend keeping the default values of items not mentioned above.
+
+#. Check or update BDF information of PCI devices of each pre-launched VM;
+   Check it in the ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``.
+
+#. Build the ACRN hypervisor and copy the artifact ``acrn.32.out`` to the
+   ``/boot`` directory:
+
+   #. Build ACRN hypervisor:
+
+      .. code-block:: none
+
+         $ make
+
+   #. Copy ``acrn.32.out`` to a removable disk.
+
+   #. Plug in the removable disk into the NUC's USB port.
+
+   #. Copy the ``acrn.32.out`` from removable disk to ``/boot`` directory.
+
+Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
+***********************************************************
+
+#. Append the following configuration to the ``/etc/grub.d/40_custom`` file:
+
+   .. code-block:: none
+
+      menuentry 'ACRN hypervisor Logical Partition Scenario' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+              recordfail
+              load_video
+              gfxmode $linux_gfx_mode
+              insmod gzio
+              insmod part_gpt
+              insmod ext2
+
+              echo 'Loading hypervisor logical partition scenario ...'
+              multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+              module /boot/bzImage
+      }
+
+   .. note:: 
+
+      The VMs kernel cmdline is defined in head file
+      ``hypervisor/scenarios/logical_partition/vm_configurations.h``
+
+      In the head file
+      ``hypervisor/scenarios/logical_partition/vm_configurations.h``, the
+      ``VMx_CONFIG_OS_BOOTARG_*`` macros are the only ones to use bootarg in its
+      singular form. The rest of the code uses bootargs (plural).
```

Comment:
This was meant as a comment about the code, i.e. it would be better that we change the name of the macros in our code. I wouldn't include this in the tutorial, just indicate which macros people need to edit if they want to change the kernel command-line parameters that are used by default.

Reply:
fixed at next version